### DISCUSSION TITLE
Limit file uploads to image types in image interface

### DISCUSSION THREAD

**Discussion by dsheiko** [directus/directus#12790] [Feature Requests]
As an admin I have PICTURE field of type image, interface image and view image.
I drag and drop a media (.mp4) file and the field accepts it.

Expected behavior: it shall block and warn the user the file type if wrong (restricted)

Project details
Directus Version: 9.0.0-rc.64
Environment: production
OS: Ubuntu 18.04.2 LTS
Node: 11.15.1

![d201](https://user-images.githubusercontent.com/1393963/117951575-96768680-b314-11eb-9a4f-ea84ca79c41e.PNG)
![d202](https://user-images.githubusercontent.com/1393963/117951588-99717700-b314-11eb-94eb-d8e3b7afc236.PNG)
![d203](https://user-images.githubusercontent.com/1393963/117951599-9bd3d100-b314-11eb-8d39-db4db3e6b3d2.PNG)
![d204](https://user-images.githubusercontent.com/1393963/117951615-9ecec180-b314-11eb-9bbd-a38619c82862.PNG)

**Comment by azrikahar** [comment]
Opting to close this in favor of #11733 which encapsulates this feature!
